### PR TITLE
Create consistent URL structure for EFM and MKT

### PR DESCRIPTION
**Pull request by dwicinas**
Revised installing topics for EFM to create a prototype URL structure.

For steps needed to replicate process, see https://docs.google.com/spreadsheets/d/1RGBluGN6Egp4DwlW-dCXXNUyjtudF0GyzbvfwqY8rfA/edit#gid=587451029

Added redirects for all install topics, but they will get written over if we start to automatically generate topics.

### CODE REVIEW

File: product_docs/docs/efm/4/installing/index.mdx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+title: "Installing Failover Manager on Linux"
+navTitle: "Installing"
+redirects:
+ - ../efm_user/03_installing_efm
+ - 13_initial_config.mdx
```
Comment (drothery-edb) [inline product_docs/docs/efm/4/installing/index.mdx:6]:
```suggestion
 - 13_initial_config
```

**Comment by drothery-edb** [comment]
@josh-heyer do you think "deb" is descriptive enough for the Debian topics in the URL? All the other OSs we spell out more.

**Comment by josh-heyer** [comment]
Let's be consistent then, @drothery-edb

**Comment by dwicinas** [comment]
@drothery-edb I think this PR, which implements standard URLs for EFM and MTK, is ready to be merged.

File: product_docs/docs/efm/4/supported_platforms.mdx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+title: "Supported platforms"
+---
+
+For information about the platforms and versions supported by PEM, see [Platform Compatibility](https://www.enterprisedb.com/platform-compatibility#pem).
```
Comment (drothery-edb) [inline product_docs/docs/efm/4/supported_platforms.mdx:5]:
```suggestion
For information about the platforms and versions supported by Failover Manager, see [Platform Compatibility](https://www.enterprisedb.com/platform-compatibility#efm).
```

**Pull request merged**